src/Bicep.LangServer.IntegrationTests/TelemetryTests.cs

```diff
@@ -359,6 +392,26 @@ public async Task BicepFileOpen_ShouldFireTelemetryEvent()
 
             bicepTelemetryEvent.EventName.Should().Be(TelemetryConstants.EventNames.LinterRuleStateOnBicepFileOpen);
             bicepTelemetryEvent.Properties.Should().Equal(properties);
+
+            properties = new Dictionary<string, string>
```

Comment:
Likewise, please at some point do a filter so you only include a known set of linter codes otherwise again we'll have to update the test when we add new rules.  Thx.

Reply:
Adding a new linter rule should require only baseline updates, no code changes.